app/src/main/java/bav/anyway/FirstScreenFragment.kt

```diff
@@ -0,0 +1,38 @@
+package bav.anyway
+
+import android.content.Context
+import android.net.Uri
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.Toast
+import androidx.navigation.Navigation
+import androidx.navigation.findNavController
+import androidx.navigation.fragment.findNavController
+import bav.anyway.ui.base.BaseFragment
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.fragment_first_screen.*
+import kotlinx.android.synthetic.main.fragment_map.*
+import javax.inject.Inject
+
```

Comment:
Empty line. Later we definitely need to implement some code style hook (whether Gradle or Git), but for now just don't place two empty lines one after another.